sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -1299,7 +1299,8 @@ private[hive] object HiveClientImpl extends Logging {
     // 3: we set all entries in config to this hiveConf.
     val confMap = (hadoopConf.iterator().asScala.map(kv => kv.getKey -> kv.getValue) ++
       sparkConf.getAll.toMap ++ extraConfig).toMap
-    confMap.foreach { case (k, v) => hiveConf.set(k, v) }
+    val fromSpark = "Set by Spark"
```

Comment:
Although this is tricky, let's collect this into the same place (https://github.com/apache/spark/pull/38084/files#r989713735).

Reply:
Gentle ping about this leftover.

Reply:
Since this is non-test code string, we should have in the same place with the others.

Reply:
yes, what is the solution there? Even when exported with private[spark] it wasn't picked up

Reply:
> yes, what is the solution there? Even when exported with private[spark] it wasn't picked up

Okay, let me make a PR to you.